cpp/src/arrow/compute/kernels/scalar_set_lookup_benchmark.cc

```diff
@@ -56,8 +56,8 @@ static void SetLookupBenchmarkString(benchmark::State& state,
 template <typename Type>
 static void SetLookupBenchmarkNumeric(benchmark::State& state,
                                       const std::string& func_name,
-                                      const int64_t value_set_length) {
-  const int64_t array_length = 1 << 18;
+                                      const int64_t value_set_length,
+                                      const int64_t array_length) {
```

Comment:
perhaps add `state.counters["value_set_length"] = value_set_length` somewhere below to make results more readable? (same in the other bench function)